src/components/BlockDocumentsTable.vue

```diff
@@ -1,12 +1,15 @@
 <template>
   <div class="block-documents-table">
     <div class="block-documents-table__filters">
-      <ResultsCount label="Block" :count="total" class="block-documents-table__results" />
+      <span class="block-documents-table__results">
+        <ResultsCount label="Block" :count="total" />
+        <BlocksDeleteButton v-if="selectedBlockDocuments.length > 0" class="block-documents-table__delete" :selected="selectedBlockDocuments.map(blockDocument => blockDocument.id)" small @delete="onDelete" />
```

Comment:
A lot of other views have [this](https://github.com/PrefectHQ/prefect-ui-library/blob/55b1089a9ff5a82be66cd67f607ae97339d1cfa6/src/components/DeploymentsList.vue#L5-L6) so that the currently selected count swaps in over the total results count when some are selected